ingestfile

```diff
@@ -49,7 +49,7 @@ while getopts ":F:pnheiu:I:m:" OPT ; do
         p) PRESERVATION_MODE="Y" ; DELIVERYTOOMNEON="N" ; DELIVERACCESSCOPIES="N" ; DELIVERYTOAIPSTORAGE="N" ;;
         n) DELIVERYTOOMNEON="N" ; DELIVERACCESSCOPIES="N" ;;
         i) DELIVERYTOOMNEON="Y" ; DELIVERACCESSCOPIES="N" ; DELIVERYTOAIPSTORAGE="Y" ;;
-        u) OPERATOR="${OPTARG}" ;;
+        u) SILENT_RUNNING="Y" ; OPERATOR="${OPTARG}" ;;
```

Comment:
Seems presumptuous to activate silent running if an operator is declared. I'd suggest a separate argument to enable silent running.